static_src/app.jsx

```diff
@@ -81,7 +81,7 @@ export default class App extends React.Component {
   }
 }
 App.propTypes = {
-  children: React.PropTypes.element,
+  children: React.PropTypes.any,
```

Comment:
This was getting rid of an annoying error because I think we pass different things as children, sometimes an arry of elements, sometimes maybe just an element?